testapps/testapp/src/main/java/com/microsoft/identity/client/testapp/MainActivity.java

```diff
@@ -264,4 +267,4 @@ public void run() {
             }
         });
     }
-}
+}
```

Comment:
Minor nit on removing the posix newline at the end of the file (and there's also some preceding newlines on this file now)